salt/modules/glance.py

```diff
@@ -104,7 +104,7 @@ def _auth(profile=None, api_version=2, **connection_args):
     Only intended to be used within glance-enabled modules
     '''
     __utils__['versions.warn_until'](
```

Comment:
I know this is not your change but why are we preferring `__opts__` instead of the direct function call?
You know the reason behind that?

Reply:
`__utils__` not `__opts__`

Reply:
Personally I thought we always preferred `__utils__` where possible since its packed in via the loader instead of importing directly. Is there a reason not to?

Reply:
Guess there really isn't a reason, I apparently just dislike it's usage, seems like mixing loader stuff with regular utils modules, but it's just my personal preference.